icu4j/maven-migration/toMaven.sh

```diff
@@ -28,6 +28,17 @@
 #
 # Then we cleanup older Eclipse project & launcher files, ant scripts, manifest.stub
 
+function safeMkdir() {
+  mkdir -p "$1"
+}
+
+function moveOrRename() {
+  SRC=$1
```

Comment:
No point in defining SRC and DST if then mv uses $1 and $1
---
But even if we use $<number> everywhere, it would be nice to add a comment to (all) the functions with the parameters (so that one knows the order).
If I'm a Windows guy I might not know if `mv` is `src dst` or `dst src`

Reply:
Done.